cloud/aws/actuators/machine/actuator_test.go

```diff
@@ -66,3 +67,43 @@ func TestCreate(t *testing.T) {
 		t.Fatalf("failed to create machine: %v", err)
 	}
 }
+
+func TestExists(t *testing.T) {
+	testCases := []struct {
+		name    string
+		exists  bool
+		machine *clusterv1.Machine
+	}{
+		{
+			name:    "machine does not exist",
+			exists:  false,
+			machine: &clusterv1.Machine{},
```

Comment:
Should there be a positive test case here as well?

Reply:
yeah, i think the pattern outlined in my other pr (https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/74) makes this a lot nicer